run_tests.sh

```diff
@@ -8,6 +8,6 @@ for i in $(find . -name go.mod -type f -print); do
   echo "running tests and lint on ${module}"
   (cd ${module} && \
     go test && \
-    golangci-lint run
+    ./bin/golangci-lint run
```

Comment:
No, this won't work. Devs run this script on their work machines and would usually have the `golangci-lint` binary in their $PATH already, not in some `./bin` sub-directory.